template.yaml

```diff
@@ -39,6 +41,12 @@ Globals:
       !If [UsePermissionsBoundary, !Ref PermissionsBoundary, !Ref AWS::NoValue]
     Runtime: nodejs16.x
     Timeout: 30
+    VpcConfig:
```

Comment:
Okay I'll check them again - it was just to keep it DRY so no big loss.

Reply:
At some point it would be nice to move this but checkov doesn't support it see https://github.com/bridgecrewio/checkov/issues/2758. I've moved them back into each lambda.